lib/elixir/lib/calendar/datetime.ex

```diff
@@ -1494,12 +1494,8 @@ defmodule DateTime do
 
   ## Examples
 
-      iex> dt1 = %DateTime{year: 2000, month: 2, day: 29, zone_abbr: "AMT",
-      ...>                 hour: 23, minute: 0, second: 7, microsecond: {0, 0},
-      ...>                 utc_offset: -14400, std_offset: 0, time_zone: "America/Manaus"}
-      iex> dt2 = %DateTime{year: 2000, month: 2, day: 29, zone_abbr: "CET",
-      ...>                 hour: 23, minute: 0, second: 7, microsecond: {0, 0},
-      ...>                 utc_offset: 3600, std_offset: 0, time_zone: "Europe/Warsaw"}
```

Comment:
These examples used different timezones, which is why they were handwritten.

However, I think your example is also useful. What about having both?

Reply:
Sounds good.